config/application.rb

```diff
@@ -59,5 +59,7 @@ class Application < Rails::Application
         resource '/api/openid_connect/userinfo', headers: :any, methods: [:get]
       end
     end
+
+    config.middleware.use Rack::Attack unless Figaro.env.disable_email_sending == 'true'
```

Comment:
What is the purpose of disabling `Rack::Attack` if email is also disabled? Is there something from the `FeatureManagement` class that we could use here that better illustrates why we're turning this off? `FeatureManagement.enable_load_testing_mode?` perhaps?

Reply:
Yeah, I had the same thought. This is what we had before, but I think it's clearer to use `enable_load_testing_mode?`. I'll change it.

Reply:
We could also add a config specifically for rate limiting? That way we could disable it without recycling which is something Joel asked for.

Reply:
Good idea. You mean without having to deploy a new build, right? All config changes require running the ASG recycle command to take effect, don't they?

Reply:
If something is behind a feature flag supposedly we can update the config on the host and run a passenger restart.